bin/apisix

```diff
@@ -609,6 +637,22 @@ local function init()
         sys_conf["worker_processes"] = "auto"
     end
 
+    if sys_conf["enable_local_dns"] then
```

Comment:
the same as https://github.com/apache/incubator-apisix/pull/1167#discussion_r388035303

Reply:
sorry, I don't know what the problem is here.  This is just to check `sys_conf["enable_local_dns"]  == true`.